tests/test-cases/Group1-Docker-Commands/1-18-Docker-Network-RM.robot

```diff
@@ -69,3 +69,17 @@ Remove network with running container
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network ls
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  test-network
+
+Add and remove network multiple times
```

Comment:
Prefer naming this - Verify user can create and delete many networks

Reply:
Its creating and deleting the same network over and over again, not creating a new network each time.